code/src/java/pcgen/persistence/lst/BioSetLoader.java

```diff
@@ -104,16 +104,8 @@ public void parseLine(LoadContext context, String lstLine, URI sourceURI)
 			ageSet.setName(colToken.nextToken());
 			while (colToken.hasMoreTokens())
 			{
-				try
-				{
-					LstUtils.processToken(context, ageSet, sourceURI,
-						colToken.nextToken());
-				}
-				catch (PersistenceLayerException e)
-				{
-					Logging.errorPrint(
-						"Error in token parse: " + e.getLocalizedMessage());
-				}
+				LstUtils.processToken(context, ageSet, sourceURI,
```

Comment:
This seems clear that instead of failing by token, one will lose the entire line if an exception is thrown - since the exception will exit the while loop

Reply:
again, the point of this PR is to remove throws declarations that never can throw exceptions and remove catch blocks that can no longer catch anything.  it can't fail by a token if an exception cannot be thrown.  i think you find if you trace the checked exception down it is declared at a low level on methods that do not throw it and that throws declaration propagates up the stack for a bit.  these exceptions cannot possibly be thrown as the code is written.